content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx

```diff
@@ -56,11 +56,15 @@ In other development environments (for example, other team members' computers):
 
 ## Baseline your production environment
 
-Assuming you have successfully transitioned to the new version of Prisma Migrate and initialized a new migration history in your repository, you now need to **baseline** any database that contains important data - such as production or staging.
+Baselining is for introducing Prisma Migrate to existing databases / brownfield projects. Concretely it's to tell Prisma Migrate to assume that one or more migration(s) were already executed so it won't try to drop and recreate existing tables.
 
-In development, we assume it is acceptable to reset your database to make sure your migration files are in sync with your database schema. This is not acceptable in a production environment, or any other environment where data must be maintained.
+Assuming you started a new migration history in your repository using `prisma migrate dev`, you now need to **baseline** any database that contains important data – such as production or staging – where the schema was not created using the `prisma migrate deploy` command.
 
-Baselining works by creating the `_prisma_migrations` table (if it does not exist) and registering each baselined migration in the `_prisma_migrations` and as "already applied". Prisma Migrate will assume that the changes contained within "already applied" migrations already exist in the database, and will not run those migrations. Any migrations that come _after_ baselined migrations will actually be applied to the database.
+In development, we assume it is acceptable to reset and seed your database to make sure your migration files are in sync with your database schema. This is not acceptable in a production environment, or any other environment where data must be maintained.
+
+You will need to use `prisma migrate resolve --applied "MIGRATION_NAME"` (see example below) to register migration(s) in the `_prisma_migrations` as "already applied", the `_prisma_migrations` table will be created if it does not exist.
+This will cause Prisma Migrate to ignore the "applied" migrations in your deployments.
+Any migrations that come _after_ baselined migrations will be applied to the database.
```

Comment:
Also for more context, I'd suggest "will be applied to the database the next time you run `prisma migrate deploy`"